### ISSUE TITLE
x86 cmovcc eflags bad recognise

### ISSUE THREAD

**Issue by jnastarot** [aquynh/capstone#1345]
when im decode some cmovcc i got in detail->x86.eflags
```cpp
X86_EFLAGS_TEST_OF | X86_EFLAGS_TEST_SF | X86_EFLAGS_TEST_ZF | X86_EFLAGS_TEST_PF | X86_EFLAGS_TEST_CF
```
for all conditions cmovcc

**Comment by aquynh** [comment]
can you fix that?

**Comment by jnastarot** [comment]
fix #1349

**Comment by aquynh** [comment]
fixed in #1349, thanks!
